newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js

```diff
@@ -44,18 +46,52 @@ const styles = {
   },
 };
 
-const AddSpriteButton = SortableElement(({ displayHint, onAdd }) => {
-  return (
-    <div style={thumbnailContainerStyle}>
-      <RaisedButton
-        onClick={onAdd}
-        label={<Trans>Add a sprite</Trans>}
-        icon={<Add />}
-        primary
-      />
-    </div>
-  );
-});
+type AddSpriteButtonProps = {|
+  onAdd: (resourceSource: ResourceSource) => void,
+  resourceManagementProps: ResourceManagementProps,
+|};
+
+// Search "activate cloud projects" in the codebase for everything to
+// remove once cloud projects are activated for the desktop app.
+const supportsCloudProjects = Window.isDev();
+
+const AddSpriteButton = SortableElement(
+  ({ onAdd, resourceManagementProps }: AddSpriteButtonProps) => {
+    return (
+      <div style={thumbnailContainerStyle}>
+        <ElementWithMenu
+          exceptionallyDontShowMenuAndFakeClickOnFirstElement={
+            !supportsCloudProjects
+          }
+          element={
+            <RaisedButton

```

Comment:
let's replace this button/elementwithmenu with a `RaisedButtonWithSplitMenu` as we did in other places.
It's clearer from a user pov.
And the default action can be the first action. (see `ResourceSelector` for similar logic)

Reply:
Done!